tokio/src/sync/mutex.rs

```diff
@@ -373,6 +372,18 @@ where
     }
 }
 
+impl<T> std::fmt::Debug for Mutex<T>
+where
+    T: std::fmt::Debug,
+{
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self.try_lock() {
+            Ok(inner) => f.debug_struct("Mutex").field("inner", &*inner).finish(),
+            Err(_) => f.debug_struct("Mutex").field("inner", &"<locked>").finish(),
```

Comment:
Nit, take it or leave it: the [standard library's implementation](https://doc.rust-lang.org/stable/src/std/sync/mutex.rs.html#397-416) calls the field for the locked value `data` rather than `inner`. We might want to do that for consistency with `std`:
```suggestion
            Ok(inner) => f.debug_struct("Mutex").field("data", &*inner).finish(),
            Err(_) => f.debug_struct("Mutex").field("data", &"<locked>").finish(),
```